### PR TITLE
DOC-7575-NerdBytes videos

### PR DESCRIPTION
**Pull request by barbnewrelic**
Added NerdByte videos to docs. Done through log parsing #37 on YouTube playlist.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence.mdx [hunk 1]
```diff
@@ -11,7 +11,13 @@ redirects:
...
+With Applied Intelligence's Proactive Detection, anomalies from your APM-monitored applications are automatically surfaced in our activity streams and anomalies feed. Each anomaly can be clicked on to bring up an automatic analysis.
+
+You can have notifications for anomalies delivered in Slack, or you can set up a webhook to deliver messages when you need them. These events are available for querying, creating custom dashboards, and alerting. By creating a Proactive Detection configuration (a group of apps you’re interested in), you can then add this configuration as a source, and have anomalies automatically correlated with other data sources via Incident Intelligence.
```
Comment (urbiz-nr) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence.mdx:16]: I'd change the second "creating". how about "By configuring Proactive Detection..."

File: src/content/docs/data-apis/manage-data/manage-data-coming-new-relic.mdx [hunk 2]
```diff
@@ -19,6 +19,12 @@ When you connect your data to New Relic, we process what we receive and apply da
   This doc is for accounts on our [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing). If you're on our original product-based pricing model, see [Original data retention](/docs/accounts/original-accounts-billing/product-based-pricing/overview-data-retention-components). Not sure which you're on? See [Overview of pricing and user model](/docs/transition-guide-our-new-pricing-plan-user-model).
 </Callout>
 
+To learn how to easily analyze the data your account is ingesting, watch this short video (3:18 minutes).
+
+<Video id="3mDiExsIjhE" type="youtube" />
+
```
Comment (urbiz-nr) [inline src/content/docs/data-apis/manage-data/manage-data-coming-new-relic.mdx:25]: SO many videos!

File: src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx [hunk 3]
```diff
@@ -9,7 +9,9 @@ metaDescription: Visualize network performance data in New Relic One.
 
 After you've set up your network data for performance monitoring, you can go to New Relic One to install several custom curated visualizations designed specifically for network performance monitoring.
 
-Once you've set up these visualizations, you'll be able to start exploring the network data your devices are reporting to New Relic One.
+Once you've set up these visualizations, you'll be able to start exploring the network data your devices are reporting to New Relic One. For some examples of how you can group and monitor your switches, routers, etc. in dashboards so you can quickly aggregate, isolate, and analyze the health of your network, watch this short video (approx. 3:45 minutes).
```
Comment (urbiz-nr) [inline src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx:12]: Is the second sentence too long?

File: src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx [hunk 4]
```diff
@@ -32,57 +34,70 @@ Once you've set up these visualizations, you'll be able to start exploring the n
...
+<figcaption>
+  Dashboard for analyzing network telemetry usage trends in New Relic One.
+</figcaption>
+
+For SNMP data, you'll see the following dashboard:
+
+![SNMP Routers and Switches](./images/dashboard_Routers_and_Switches.png 'SNMP Routers and Switches')
+
+<figcaption>
+  Dashboard for SNMP data from Router and Switch entities in New Relic One.
```
Comment (urbiz-nr) [inline src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx:77]: Router and Switch or router and switch?

**Review by urbiz-nr — APPROVED** [review]
Nice! Left a couple of comments, up to you!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence.mdx (hunk 1 above)
Comment (barbnewrelic) [reply src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence.mdx:16]: thx, more copyediting is in order!

File: src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx (hunk 3 above)
Comment (barbnewrelic) [reply src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx:12]: William Faulkner wrote it. I'll get Ernest Hemingway to edit it. :-)

File: src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx (hunk 4 above)
Comment (barbnewrelic) [reply src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx:77]: hmm... this is how it was in original text. I'll l/c it. Good catch!

File: src/content/docs/data-apis/manage-data/manage-data-coming-new-relic.mdx (hunk 2 above)
Comment (barbnewrelic) [reply src/content/docs/data-apis/manage-data/manage-data-coming-new-relic.mdx:25]: understatement :-) And there are still approx 30 more!

**Comment by barbnewrelic** [comment]
This has already been reviewed and edits made, but GH won't let me merge the PR. It doesn't need another review, just approval. Thanks!

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**